src/globals.c

```diff
@@ -158,9 +165,13 @@ static void zpc_init(void)
 	assert(rc == 0);
 	if (load_ep11_library(&ep11, true) != 0)
 		DEBUG("loading EP11 library failed");
-	else
+	else {
 		DEBUG("loaded EP11 library: %u.%u", ep11.version.major,
 	            ep11.version.minor);
+		aes_ep11 = 1;
+		if (ep11.version.major >= 3)
+			ecdsa_ep11 = 1;
```

Comment:
This is again just the EP11 host library version, but not the crypto card version.